compiler/src/dmd/backend/symbol.d

```diff
@@ -327,7 +327,7 @@ version (SCPP_HTOD)
 
 @trusted
 extern (C)
-Symbol * symbol_calloc(const(char)[] id)
+Symbol * symbol_calloc(const(char)[] id) @nogc
```

Comment:
unrelated? also why have a trailing `@nogc` with a leading `@trusted`?

Reply:
The prototype was annotated `@nogc`, but the implementation not, so importing it without this annotation gives an error.

Reply:
Ah, OK. would be nice to be consistent with the leading/trailing of attributes though.

Reply:
Done